### ISSUE TITLE
Add database backend

### ISSUE THREAD

**Issue by mt-digital** [jyoshimi/history_graph#8]

**Comment by jyoshimi** [comment]
Yes agreed.  I'd like it if the database could just sit locally in the folder that contains history_net.html, just to make it super simple to download and open this in one's browser, but not sure if there is any database system or technique  that allows this.  

Also, insofar as this is used to teach people to just use git to edit something, editing a json file is pretty easy....

**Comment by mt-digital** [comment]
Definitely. I think just using a .json is fine. We could close this if we will just go with the current solution.
